release/nightly_tests/dataset/map_batches_benchmark.py

```diff
@@ -0,0 +1,149 @@
+            num_calls = 2
+            test_name = f"map-batches-{batch_format}-{batch_size}-{num_calls}-default"
+            benchmark.run(
+                test_name,
+                map_batches,
+                input_ds=input_ds,
+                batch_format=batch_format,
+                batch_size=batch_size,
+                num_calls=num_calls,
+            )
+            test_name = f"map-batches-{batch_format}-{batch_size}-{num_calls}-lazy"
+            benchmark.run(
+                test_name,
+                map_batches,
+                input_ds=lazy_input_ds,
+                batch_format=batch_format,
+                batch_size=batch_size,
+                num_calls=num_calls,
+            )
+
+    # Test multiple calls of map_batches.
+    for num_calls in num_calls_list:
+        for compute in ["tasks", ActorPoolStrategy(1, 1)]:
+            batch_size = 4096
+            if compute == "tasks":
+                compute_strategy = "tasks"
+            else:
+                compute_strategy = "actors"
+
+            test_name = (
+                f"map-batches-{batch_format}-{batch_size}-{num_calls}-"
+                f"{compute_strategy}-default"
+            )
+            benchmark.run(
+                test_name,
+                map_batches,
+                input_ds=input_ds,
+                batch_format=batch_format,
+                batch_size=batch_size,
+                compute=compute,
+                num_calls=num_calls,
+            )
+            test_name = (
+                f"map-batches-{batch_format}-{batch_size}-{num_calls}-"
+                f"{compute_strategy}-lazy"
+            )
+            benchmark.run(
+                test_name,
+                map_batches,
+                input_ds=lazy_input_ds,
+                batch_format=batch_format,
+                batch_size=batch_size,
+                compute=compute,
+                num_calls=num_calls,
+            )
+
+    # Test different batch formats of map_batches.
+    for current_format in ["pyarrow", "pandas", "numpy"]:
+        new_input_ds = input_ds.map_batches(
+            lambda ds: ds, batch_format=current_format, batch_size=None
+        ).fully_executed()
+        for new_format in ["pyarrow", "pandas", "numpy"]:
+            for batch_size in batch_sizes:
+                test_name = f"map-batches-{current_format}-to-{new_format}-{batch_size}"
+                benchmark.run(
+                    test_name,
+                    map_batches,
+                    input_ds=new_input_ds,
+                    batch_format=new_format,
+                    batch_size=batch_size,
+                    num_calls=1,
+                )
+
+    # Test reading multiple files.
+    input_ds = ray.data.read_parquet(
+        "s3://air-example-data/ursa-labs-taxi-data/by_year/2018"
+    )
+    for batch_format in batch_formats:
+        for compute in ["tasks", "actors"]:
+            test_name = f"map-batches-{batch_format}-{compute}-default"
```

Comment:
nit: call out what is being set default? It looks "default_batch_size" in this case.

Reply:
Sorry it should not be "default", changed to "multi-files"